libtelemetry/src/full/java/com/mapbox/android/telemetry/crash/CrashReporter.java

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.android.telemetry.crash;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+
+import com.mapbox.android.core.FileUtils;
+import com.mapbox.android.telemetry.CrashEvent;
+
+import java.io.File;
+
+import static com.mapbox.android.telemetry.MapboxTelemetryConstants.MAPBOX_TELEMETRY_PACKAGE;
+
+public final class CrashReporter {
```

Comment:
I'd prefer `ErrorReporter` for this class, and we might eventually change the event name, just to avoid the 'C' word